hotel/models/hotel.py

```diff
@@ -205,13 +205,41 @@ class FolioRoomLine(models.Model):
     _description = 'Hotel Room Reservation'
     _rec_name = 'room_id'
 
+    @api.model
+    def _get_checkin_date(self):
+        if self._context.get('tz'):
+            to_zone = self._context.get('tz')
+        else:
+            to_zone = 'UTC'
```

Comment:
the above 4 lines should be written in 3 lines:
to_zone = 'UTC'
if self._context.get('tz'):
    to_zone = self._context.get('tz')

Reply:
What about this ?

`to_zone = self._context.get('tz') or self.env.user.partner_id.tz or 'UTC'`